app/controllers/concerns/partner_category_filter.rb

```diff
@@ -27,10 +28,21 @@ def apply_to(query = Partner)
   end
 
   def categories
+    # only return categories that have partners
```

Comment:
Is this a rails convention to progressively indent multi-line comments? I've not seen that before.

Reply:
nope. just a me thing i am afraid